src/components/Breadcrumbs.js

```diff
@@ -22,10 +22,17 @@ export default class Breadcrumbs extends Component {
       const paths = splat.split('/');
       links = _.map(paths, (path, i) => {
         const before = (i == 0) ? '' : paths.slice(0, i).join('/') + '/';
-        return {
-          href: `${base}/${before}${path}`,
-          label: path
-        };
+        if (type == "data files") {
```

Comment:
Actually, `/_api/data/movies/` returns result for the `/movies/` folder and `/_api/data/movies` returns for `movies.yml` file, both being valid data source in their own context and both resulting in `site.data.movies`